include/dolphin/os.h

```diff
@@ -64,6 +64,7 @@ OSTime OSGetTime();
 OSTick OSGetTick();
 void OSReport(const char* msg, ...);
 void OSPanic(const char* file, int line, const char* msg, ...);
+void OSFatal(uint a, uint b, const char* msg);
```

Comment:
Dolphin headers always use `s32`/`u32`

But OSFatal in particular looks like it's GXColor, not u32

Reply:
Changed to GXColor, but left constructing these in ErrorHandler for later since it gets weird